megazords/lockbox/android/build.gradle

```diff
@@ -77,6 +79,21 @@ cargo {
     exec = rootProject.ext.cargoExec
 }
 
+nativeLibs {
+    nss {
+        lib "libplc4.*"
+        lib "libplds4.*"
+        lib "libnspr4.*"
+        lib "*freebl3.*"
+        lib "*nss3.*"
+        lib "*nssckbi.*"
+        lib "*nssutil3.*"
+        lib "*smime3.*"
+        lib "*softokn3.*"
+        // lib "libssl3.*"
+    }
+}
```

Comment:
It's a shame we have to duplicate this list in several places, is there any way we could arrange for it to be shared?